### ISSUE TITLE
Incompatible api with accounting.js

### ISSUE THREAD

**Issue by tiemevanveen** [nashdot/accounting-js#8]
> Same as original accounting.js, but .. 

Not 100%, The settings api is different.

The docs [here](http://openexchangerates.github.io/accounting.js/#documentation) say you could set this different defaults in `accounting.settings.currency` if you like. This is a bit confusing as the api for setting these defaults in this package is different.

Accounting.js:

```
accounting.settings = {
	currency: {
		symbol : "$",   // default currency symbol is '$'
		format: "%s%v", // controls output: %s = symbol, %v = value/number (can be object: see below)
		decimal : ".",  // decimal point separator
		thousand: ",",  // thousands separator
		precision : 2   // decimal places
	},
	number: {
		precision : 0,  // default precision on numbers is 0
		thousand: ",",
		decimal : "."
	}
}
```

vs this package:

```
accounting.settings = {
  symbol: '$',        // default currency symbol is '$'
  format: '%s%v',     // controls output: %s = symbol, %v = value (can be object, see docs)
  decimal: '.',       // decimal point separator
  thousand: ',',      // thousands separator
  precision: 2,       // decimal places
  grouping: 3,        // digit grouping (not implemented yet)
  stripZeros: false,  // strip insignificant zeros from decimal part
  fallback: 0         // value returned on unformat() failure
};
```

I've also found that setting the whole settings object at once does not work.. see https://github.com/nashdot/accounting-js/pull/7

**Comment by STAH** [comment]
I can rollback to original version with nested settings object but what the real use case to have different settings for numbers and currencies?

**Comment by STAH** [comment]
Also, in case of rollback, should we add `common` section with additional settings for `stripZeros` and `fallback`?

**Comment by tiemevanveen** [comment]
I don't think that it is a problem that the packages differ, as long as it's clear from the documentation. I blindly followed the docs because the readme said that the package is the same. It fine to differ from the original api as long as it's documented right ? And since this is breaking you might also want to bump the major version number so users know that the api is different. 

That way users won't do `accounting.settings.currency = ... ` for example.

**Comment by STAH** [comment]
Understand... Need create new documentation and demo site (up to date and showing new features) :)
Besides, i introduced another breaking change in original code: unformat is not more called from formatXxx methods (respecting consistency). So browser version could be broken...

**Comment by tiemevanveen** [comment]
Check.

**Comment by tiemevanveen** [comment]
Hey @STAH any updates on this one? Would it still be possible for you to write some docs for the current api of the npm accounting-js package?
